src/metabase/analytics/stats.clj

```diff
@@ -375,7 +375,8 @@
   ([summary execution]
    (-> summary
        (update :executions u/safe-inc)
-       (update-in [:by_status (if (:has_error execution)
+       ;; MYSQL is returning true as 1, false as 0! what!
```

Comment:
hahaha Chris is fighting with this too 🤣

Reply:
:(

Reply:
I also had a more complex boolean expression return `0M` (a bigdecimal) from MariaDB. FML 😆